docs/stacks-guide-aws.md

```diff
@@ -1,54 +1,56 @@
 ---
-title: "Crossplane Stacks Guide: AWS Setup"
+title: "Stacks Guide: AWS Setup"
 toc: true
-weight: 332
+weight: 411
 indent: true
 ---
 
-# Crossplane Stacks Guide: AWS Setup
+# Stacks Guide: AWS Setup
 
 ## Table of Contents
 
-  1. [Crossplane Stacks Guide: AWS Setup](#crossplane-stacks-guide-aws-setup)
-  1. [Table of Contents](#table-of-contents)
   1. [Introduction](#introduction)
-  1. [Before you begin](#before-you-begin)
-  1. [Install AWS stack](#install-aws-stack)
+  1. [Install the AWS stack](#install-the-aws-stack)
   1. [Configure the AWS account](#configure-the-aws-account)
   1. [Configure Crossplane Provider for
     AWS](#configure-crossplane-provider-for-aws)
   1. [Set Up Network Configuration](#set-up-network-configuration)
   1. [Configure Provider Resources](#configure-provider-resources)
   1. [Recap](#recap)
+  1. [Next Steps](#next-steps)
 
 ## Introduction
 
-In this guide we are focusing on WordPress stack, and will show how it could be
-deployed using AWS resources. WordPress stack uses a generic Kubernetes cluster
-and a MySQL database as its resource, which are not tied to a specific provider
-(also known as **resource claims**). Instead, these resource claims could be
-configured to use a be satisfied by a specific cloud provider.
+In this guide, we will set up a GCP provider in Crossplane so that we
```

Comment:
this is the AWS guide, not the GCP guide

Reply:
Fixed